packages/solid/components/Badge/Badge.tsx

```diff
@@ -51,20 +39,24 @@ const BadgeContainer: Component<BadgeProps> = props => {
 };
 
 const Badge: Component<BadgeProps> = (props: BadgeProps) => {
-  const tone = createMemo(() => getTone(props));
-  const title = createMemo(() => getTitle(props.title));
+  const tone = createMemo(() => props.tone ?? styles.tone);
 
   return (
-    <BadgeContainer {...props} padding={props.padding} tone={tone()} style={props.style}>
+    <BadgeContainer
+      {...props}
+      height={props.height || styles.Text.base.lineHeight}
+      tone={tone()}
+      style={props.style}
+    >
       <Text
         color={props.textColor}
+        lineHeight={(props.height || styles.Text.base.lineHeight) + 2}
```

Comment:
if this `2` is correcting for the border width, we should reference `styles.Container.base.border.width`